contrib/pg_tde/documentation/docs/test.md

```diff
@@ -36,11 +36,11 @@ Here's how to do it:
 3. Rotate the principal key when needed:
 
     ```
-    SELECT pg_tde_rotate_principal_key(); -- uses automatic key versionin
+    SELECT pg_tde_rotate_key(); -- uses automatic key versionin
     -- or
-    SELECT pg_tde_rotate_principal_key('new-principal-key', NULL); -- specify new key name
+    SELECT pg_tde_rotate_key('new-key', NULL); -- specify new key name
     -- or
-    SELECT pg_tde_rotate_principal_key('new-principal-key', 'new-provider'); -- changeprovider
+    SELECT pg_tde_rotate_key('new-key', 'new-provider'); -- changeprovider
```

Comment:
This function does not exist, but I could remove it as a separate PR unless you want to remove it yourself.

Reply:
Pointed to key management functions description from this section